packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -106,8 +106,8 @@ export const TokensList: React.FC<TokensListProps> = props => {
                   <thead>
                     <Tr>
                       <Th>Token</Th>
-                      <Th width="250px">Value</Th>
-                      <Th width="250px">Example</Th>
+                      <Th style={{width: '250px'}}>Value</Th>
```

Comment:
I realized we could revert the changes we made to the table component and use an inline-style instead.